docs/infra/set-up-aws-account.md

```diff
@@ -0,0 +1,65 @@
+# Set up AWS account
```

Comment:
Are people supposed to follow this guide or run the `bin/set-up-account.sh` script / `set-up-account` make command? (I think I'm a little confused about when that script is used)

Reply:
Similarly for `docs/infra/set-up-app.md` vs. `bin/set-up-app-backends.sh`, and the destroy docs vs. the destroy script.

Are the docs essentially just documenting the manual process if that needs to happen? Or are the scripts only intended for template-repo CI testing?

Reply:
Good question, and the fact that it's not super clear probably reveals my own indecisiveness about it.

For now I'm thinking that people are expected to do the manual process, and the scripts are only for template-repo CI testing. I'm not yet confident that the scripts are general purpose enough and robust enough for users of the template. In particular, I'm not sure we have good error handling, documentation on parameters, etc., and with the `-auto-approve` steps and automatically overwriting local files, I'm worried it might be a little dangerous and surprise people.

Reply:
Got it, that makes sense. I wonder if we can make that more explicit, e.g. placing the scripts in a template-only folder or adding a disclaimer at the top of those files.